### DISCUSSION TITLE
I think antivirus deleted a Copilot's file and now Copilot doesn't work

### DISCUSSION THREAD

**Discussion by pagmat-dev1** [community/community#19552] [Copilot Conversations]
Hi everybody,

Last week, after accepting new terms and conditions with an update. Kaspersky Endpoint deleted a Copilot file and then I received this error:

Your task failed with the message: Failed to start host 'node.x86$C94B8CFE-E3FD-4BAF-A941-2866DBB566FE' Host exited with code '1'. (ErrorKind:Error)

What should I do?

**Comment by melvincwng** [comment]
@pagmat-dev1, would reinstalling GitHub Copilot help to resolve this issue?

**Reply by pagmat-dev1** [reply]
I tried to reinstall Github Copilot but still same.

**Reply by melvincwng** [reply]
@pagmat-dev1, can you try the following steps:

1. Uninstall GitHub Copilot
2. Log out of your GitHub account in whichever IDE you are using (either VS, VS Code, Neovim, Jetbrains IDE)
3. Close your IDE code editor
4. Open your IDE code editor back
5. Log in back to your GitHub account (please kindly make sure you are authorized to use GitHub Copilot either free or if you have a subscription)
6. This should regenerate the auth token from GitHub servers.
7. Reinstall the Copilot extension
8. Try re-running GitHub Copilot again on your IDE.

Hopefully these steps work for you.

If this is helpful, please kindly 'Mark as answer'. 

Else, hopefully another member of the community would be able to help resolve your issue.

All the best 👍🏻

**Reply by pagmat-dev1** [reply]
I did every steps one by one, unfortunately it still doesn't work. :(
GitHub Copilot is free until August 22 for me btw

**Comment by D7EAD** [comment]
Hi! 

If your anti-virus software detected a false-positive in Copilot's files, I would advise the following:
1. Entirely remove Copilot and any associated files. Make note of where it is installed.
2. Add an exception in your anti-virus software to not scan Copilot's files. 
3. Reinstall Copilot and complete the setup process. If at any point your anti-virus still detects a false positive, you _should_ be able to restore the deleted/quarantined files from within your anti-virus' management application. 

Hopefully this helps! If so, please kindly **Mark as Answer** so others facing the same problem can find the proper resolution.

**Reply by pagmat-dev1** [reply]
I am not a security expert. I cant take the risk to add an exception even for Github Copilot. If its a false-positive I would prefer to wait Kaspersky fix this.

**Reply by D7EAD** [reply]
If you did a fresh reinstall of Github Copilot, and if the files were still detected as bad eggs by Kaspersky, are you able to whitelist the specific files in question? Most modern anti-virus software allow whitelisting of specific files, rather than full exceptions of directories. 

Since you seem understandably hesitant to add an exception to Copilot's installation location, have you considered whitelisting the specific files in question? If you'd rather not, I believe waiting for Kaspersky to address this would be your prime choice.

**Reply by pagmat-dev1** [reply]
Yes I did a fresh reinstall of Github Copilot, and interestingly Kaspersky did not detected Copilot's files as a virus after reinstallition. Also, I am able to add specific files and folders to the whitelist but I would not prefer this yet.

**Reply by D7EAD** [reply]
Fair enough, hopefully you're able to make use of Copilot soon!
